### DISCUSSION TITLE
Use `"moduleResolution": "Bundler"` in the Vike project

### DISCUSSION THREAD

**Discussion by rtritto** [vikejs/vike#1953] [Discussion]
[/vike/tsconfig.json](https://github.com/vikejs/vike/blob/main/vike/tsconfig.json)

This allows to omit the file extension in the imports of a `.ts` file.

Example for `tsconfig.json`:
```json
{
  "compilerOptions": {
    "allowImportingTsExtensions": true,
    "allowSyntheticDefaultImports": true,
    "esModuleInterop": true,
    "forceConsistentCasingInFileNames": true,
    "isolatedModules": true,
    "module": "ESNext",
    "moduleResolution": "Bundler",
    "noEmit": true,
    "resolveJsonModule": true,
    "skipLibCheck": true,
    "strict": true,
    "target": "ESNext"
  }
}
```

**Comment by brillout** [comment]
In general I think it's best we focus contributions to be about feature requests or bugs. That said, if you can make a PR that keeps the CI green then why not :+1:

**Comment by rtritto** [comment]
Thanks, I opened #1955
